spike/CodeReaderClass.py

```diff
@@ -0,0 +1,151 @@
+class CodeParser():
+
+    #Parsing variables
+    PORT = ''
+    OPERATION = ''
+    NUMBER = ''
+
+    #Parsing libraries (or "keywords" if you wanna not sound like an arrogant prick)
+    OPERATIONS = ['+', '-', '*', '/', '=']
+    NUMBERS = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9']
+    WHITESPACE = ' '
+    PORTS = ["THROTTLE", "TURN"]
+
+    #Initialize the CodeParser
+    def __init__(self):
+        self.PATH = "Racer.txt"
+        self.SPEED = 0
+        self.ANGLE = 0
+        self.analyzer()
```

Comment:
Don't call `analyzer`. We can always initialize and call the analyzer immediately after if we need to but calling it here will back us into a corner of always doing it at the same time.